catalog/transformer/catalog-transformer-geojson-metacard/pom.xml

```diff
@@ -46,6 +45,9 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Embed-Dependency>
+                            catalog-core-api-impl;scope=!test
```

Comment:
Because it does not export its classes. This page helps explain the reasoning behind only exporting API interfaces. http://wiki.osgi.org/wiki/Export_Only_APIs